Modules/Relationships/src/Relationships.Application/ApplicationErrors.cs

```diff
@@ -6,10 +6,10 @@ public static class ApplicationErrors
 {
     public static class Relationship
     {
-        public static ApplicationError PeerIsToBeDeleted(string peerToBeDeleted)
+        public static ApplicationError PeerIsToBeDeleted()
         {
             return new ApplicationError("error.platform.validation.relationship.peerIsToBeDeleted",
-                $"Cannot establish relationship with '{peerToBeDeleted}' because they are in status 'ToBeDeleted'.");
+                "Cannot establish relationship with the owner of the template because they are in status 'ToBeDeleted'.");
```

Comment:
I would say `...because the owner...` instead.

I am aware of modern tendencies in English language, however, despite being highly skilled in the use of it, it still sounds weird when people use "they" for a single person.

This is however an aesthetical concern so you may do however you think is best.